js/Heap.js

```diff
@@ -15,29 +16,32 @@ import Package from 'react-native-package';
  * https://github.com/negativetwelve/react-native-package
  */
 export default Package.create({
-  json: require('../package.json'),
+  json: require("../package.json"),
   nativeModule: NativeModules.RNHeap,
   enabled: Platform.select({
     ios: true,
+    android: true
   }),
-  export: (Heap) => ({
+  export: Heap => ({
     // App Properties
-    setAppId: (appId) => Heap.setAppId(appId),
+    setAppId: appId => Heap.setAppId(appId),
 
     // User Properties
-    identify: (identity) => Heap.identify(identity),
-    addUserProperties: (properties) => Heap.addUserProperties(properties),
+    identify: identity => Heap.identify(identity),
+    addUserProperties: properties => Heap.addUserProperties(properties),
 
     // Event Properties
-    addEventProperties: (properties) => Heap.addEventProperties(properties),
-    removeEventProperty: (property) => Heap.removeEventProperty(property),
+    addEventProperties: properties => Heap.addEventProperties(properties),
+    removeEventProperty: property => Heap.removeEventProperty(property),
```

Comment:
Have to add a `clean` function to eliminate arrays/maps from what's sent to AEP, AUP, and track.